### DISCUSSION TITLE
Help Please: Failed to apply patch

### DISCUSSION THREAD

**Discussion by rocketedaway** [chromiecraft/chromiecraft#2787] [General]
Hi,

I just downloaded the client maybe 1 - weeks ago. I tried logging in and I get sent to the download screen. It sits there for a while saying 0% downloaded. Then it says I need to restart the client to apply the patch. After clicking restart I am taken to the login screen and a message says it failed to apply patch.

I tired googling and I found nothing. Should I re-download the client?

Any help would be appreciated. Thanks!

**Comment by 55Honey** [comment]
Hi @rocketedaway 
Apparently you try to login with your Email. Please use your username instead.

Consider joining our discord for even better community interaction :-)

**Reply by rocketedaway** [reply]
Thanks :) Ill give it a try!

**Comment by HugginsHomes** [comment]
This solved it for me.  Thank you.
